tailwind.config.mjs

```diff
@@ -15,7 +15,7 @@ export default {
       black: '#000',
     },
     fontSize: {
-      sm: ['16px', '20px'],
+      sm: ['14px', '20px'],
       base: ['18px', '24px'],
```

Comment:
Etkö puhunu joskus et sm tai base ois aina 16?

Reply:
Hyvä huomio! Figman designissa oli leipäteksti 18px, joten otin sen suoraan tänne tailwindiinkin

Meillä on näemmä 18px font-size myös main-elementille, katsoin layout.astro filestä. Käytännössä tää siis vaikuttaa siten, että kaikki rem-yksiköt (joita on best practice käyttää, ja näin myös tailwind tekee) on suhteutettu tohon. 1rem = 18px.

Tailwind näyttää olettavan että 1rem = 16px ainakin dokumentaation puolesta kyllä 🤔